domain/src/main/java/me/androidbox/domain/authentication/model/Event.kt

```diff
@@ -1,14 +1,14 @@
 package me.androidbox.domain.authentication.model
 
 data class Event(
-    val id: String,
-    val title: String,
-    val description: String,
-    val startDateTime: Long,
-    val endDateTime: Long,
-    val remindAt: Long,
-    val eventCreatorId: String,
-    val isUserEventCreator: Boolean,
-    val attendees: String,
-    val photos: String,
+    val id: String = "",
+    val title: String = "",
+    val description: String = "",
+    val startDateTime: Long = 0L,
+    val endDateTime: Long = 0L,
+    val remindAt: Long = 0L,
+    val eventCreatorId: String = "",
+    val isUserEventCreator: Boolean = false,
+    val attendees: String = "",
+    val photos: String = "",
```

Comment:
What's the use of adding default parameters here?

You would never need an empty Event

Reply:
@MKiperszmid the reason I added default parameters because I set this state in the EventScreenState. What should I be doing?

Reply:
@MKiperszmid Removed the default parameters